src/Native/System.Native/pal_errno.h

```diff
@@ -26,94 +26,94 @@
-    PAL_ENETDOWN         = 0x10026,           // Network is down.
-    PAL_ENETRESET        = 0x10027,           // Connection aborted by network.
-    PAL_ENETUNREACH      = 0x10028,           // Network unreachable.
-    PAL_ENFILE           = 0x10029,           // Too many files open in system.
-    PAL_ENOBUFS          = 0x1002A,           // No buffer space available.
-    PAL_ENODEV           = 0x1002C,           // No such device.
-    PAL_ENOENT           = 0x1002D,           // No such file or directory.
-    PAL_ENOEXEC          = 0x1002E,           // Executable file format error.
-    PAL_ENOLCK           = 0x1002F,           // No locks available.
-    PAL_ENOLINK          = 0x10030,           // Reserved.
-    PAL_ENOMEM           = 0x10031,           // Not enough space.
-    PAL_ENOMSG           = 0x10032,           // No message of the desired type.
-    PAL_ENOPROTOOPT      = 0x10033,           // Protocol not available.
-    PAL_ENOSPC           = 0x10034,           // No space left on device.
-    PAL_ENOSYS           = 0x10037,           // Function not supported.
-    PAL_ENOTCONN         = 0x10038,           // The socket is not connected.
-    PAL_ENOTDIR          = 0x10039,           // Not a directory or a symbolic link to a directory.
-    PAL_ENOTEMPTY        = 0x1003A,           // Directory not empty.
-    PAL_ENOTRECOVERABLE  = 0x1003B,           // State not recoverable.
-    PAL_ENOTSOCK         = 0x1003C,           // Not a socket.
-    PAL_ENOTSUP          = 0x1003D,           // Not supported (same value as EOPNOTSUP).
-    PAL_ENOTTY           = 0x1003E,           // Inappropriate I/O control operation.
-    PAL_ENXIO            = 0x1003F,           // No such device or address.
-    PAL_EOVERFLOW        = 0x10040,           // Value too large to be stored in data type.
-    PAL_EOWNERDEAD       = 0x10041,           // Previous owner died.
-    PAL_EPERM            = 0x10042,           // Operation not permitted.
-    PAL_EPIPE            = 0x10043,           // Broken pipe.
-    PAL_EPROTO           = 0x10044,           // Protocol error.
-    PAL_EPROTONOSUPPORT  = 0x10045,           // Protocol not supported.
-    PAL_EPROTOTYPE       = 0x10046,           // Protocol wrong type for socket.
-    PAL_ERANGE           = 0x10047,           // Result too large.
-    PAL_EROFS            = 0x10048,           // Read-only file system.
-    PAL_ESPIPE           = 0x10049,           // Invalid seek.
-    PAL_ESRCH            = 0x1004A,           // No such process.
-    PAL_ESTALE           = 0x1004B,           // Reserved.
-    PAL_ETIMEDOUT        = 0x1004D,           // Connection timed out.
-    PAL_ETXTBSY          = 0x1004E,           // Text file busy.
-    PAL_EXDEV            = 0x1004F,           // Cross-device link.
+    PAL_E2BIG = 0x10001,           // Argument list too long.
+    PAL_EACCES = 0x10002,          // Permission denied.
+    PAL_EADDRINUSE = 0x10003,      // Address in use.
```

Comment:
I went back and forth about this one...I decided against it because this flag is only available on 3.7+ but clang-format-3.6 is the default on Ubuntu 14 and for the version of llvm that we currently support. Once 3.7 and 3.8 are common place we can upgrade and add that flag, but for now I think we should keep it out so we can stay with the most-easily-accessible version